### PR TITLE
add task icons to right panel

### PR DESCRIPTION
**Pull request by Moe-Shoman**

### CODE REVIEW

File: src/components/Content/RightPanel/TaskDetails/TaskIcons/TaskIcons.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  }
+
+  return (
+    <div className="task-icons">
+      <OverlayTrigger placement="bottom" overlay={getTooltip('chat')}>
+        <FaCommentsO size={36} className="icon"/>
+      </OverlayTrigger>
+
+      <OverlayTrigger placement="bottom" overlay={getTooltip('video-chat')}>
+        <a href={jitsiRoom + gid} target="_self" rel="nofollow noreferrer noopener">
```
Comment (elimisteve) [inline src/components/Content/RightPanel/TaskDetails/TaskIcons/TaskIcons.js]: @Moe-Shoman What does `target="_self"` do?

Comment (elimisteve) [reply src/components/Content/RightPanel/TaskDetails/TaskIcons/TaskIcons.js]: Does `tarket="_blank"` make more sense? Hmm...

Comment (Moe-Shoman) [reply src/components/Content/RightPanel/TaskDetails/TaskIcons/TaskIcons.js]: `_self` intends to load the document in the same frame.

Comment (elimisteve) [reply src/components/Content/RightPanel/TaskDetails/TaskIcons/TaskIcons.js]: Okay, can you do `target="_blank"` for now and we'll mess with the popup thing later?

Comment (Moe-Shoman) [reply src/components/Content/RightPanel/TaskDetails/TaskIcons/TaskIcons.js]: Yup, just pushed the change

**Pull request merged**